### DISCUSSION TITLE
onChange callback doesn't contain event property

### DISCUSSION THREAD

**Discussion by haond10adp** [mskocik/svelecte#254] [Q&A]

**Comment by mskocik** [comment]
And why would you need that?

**Comment by haond10adp** [comment]
I need to access the containing form to call submit programmatically `event.currentTarget.form.requestSubmit()`

**Comment by mskocik** [comment]
When you set `name` property, `<select>` is created. You can attach `change` listener to it directly.

**Comment by haond10adp** [comment]
Can you show me the code?

**Comment by mskocik** [comment]
I am on mobile only this week. But in short in onChange handler get your form by id and call your submit function

**Comment by haond10adp** [comment]
:+1:  Have a good holiday. I'm waiting for your code next week
